src/Clearinghouse.sol

```diff
@@ -363,7 +368,9 @@ contract Clearinghouse is Policy, RolesConsumer, CoolerCallback {
             });
         }
 
+        // Defund and log the event
         token_.transfer(address(TRSRY), amount_);
+        emit Defund();
```

Comment:
Same with this, emit the token (since it can be DAI or sDAI) and amount (negative number) as parameters.